src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const logo = document.querySelector('.logo');
+
+logo.addEventListener('click', function () {
+  const promise1 = new Promise((resolve) => {
+    resolve();
+  });
+
+  promise1.then(() => {
+    createMessage('Promise was resolved', 'message');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  reject(new Error('Error'));
+});
+
+promise2.catch(() => {
```

Comment:
Consider logging the error message in the `catch` block to provide more information about the error. This can be done by passing the error object to the `catch` function and logging it inside the block.